### PR TITLE
added questions

### PR DESCRIPTION
**Pull request by mickaelandrieu**

### CODE REVIEW

File: data/bundles.yml [hunk 1]
```diff
@@ -63,3 +63,8 @@ questions:
             - {value: "$container->get('kernel')->getBundles()",          correct: true}
             - {value: "$container->getParameter('bundles')",              correct: false}
             - {value: "$container->getParameter('application.bundles')",  correct: false}
+    -
+        question: 'True or False ? the entry point of a Bundle MUST extends `Symfony\HttpKernel\Bundle\Bundle` class.'
```
Comment (xabbuh) [inline data/bundles.yml]: Isn't it technically sufficient enough to implement the `BundleInterface`?

File: data/command-line.yml [hunk 2]
```diff
@@ -123,3 +123,18 @@ questions:
...
             - {value: php app/console doctrine:fixtures,        correct: false}
             - {value: php app/console doctrine:fixtures:import, correct: false}
+    -
+        question: 'Symfony Console: which one(s) are valid verbosity levels ?'
+        answers:
+            - {value: OutputInterface::VERBOSITY_QUIET,           correct: true}
+            - {value: OutputInterface::VERBOSITY_NORMAL,          correct: true}
+            - {value: OutputInterface::VERBOSITY_VERBOSE,         correct: true}
+            - {value: OutputInterface::VERBOSITY_VERY_VERBOSE,    correct: true}
+            - {value: OutputInterface::DEBUG,                     correct: true}
```
Comment (xabbuh) [inline data/command-line.yml]: must be `OutputInterface::VERBOSITY_DEBUG`

File: data/command-line.yml [hunk 3]
```diff
@@ -123,3 +123,18 @@ questions:
...
+    -
+        question: 'Symfony Console: InputArgument::REQUIRED, InputArgument::OPTIONAL and ... ?'
+        answers:
+            - {value: InputArgument::DEFAULT,    correct: false}
+            - {value: OutputInterface::ARRAY,    correct: true}
```
Comment (xabbuh) [inline data/command-line.yml]: There is no `ARRAY` constant in the `OutputInterface`.

Comment (xabbuh) [inline data/command-line.yml]: You probably wanted to use `InputArgument::IS_ARRAY`.

File: data/misc.yml [hunk 4]
```diff
@@ -35,3 +35,16 @@ questions:
             - {value: public static function create($errorReportingLevel = null, $displayErrors = true), correct: false}
             - {value: public static function load($errorReportingLevel = null, $displayErrors = true),   correct: false}
             - {value: public static function start($errorReportingLevel = null, $displayErrors = true),  correct: false}
+    -
+        question: 'In order to be able to use render_hinclude(url(...)), we need to add this configuration in ``framework` section:'
+        answers:
+            - {value: 'hinclude: enabled',             correct: false}
+            - {value: 'fragments: {path:/_fragment}',  correct: false}
+            - {value: 'none of the above',             correct: false}
```
Comment (xabbuh) [inline data/misc.yml]: There should be one valid answer.

File: data/bundles.yml (hunk 1 above)
Comment (mickaelandrieu) [inline data/bundles.yml]: Agree,  but the docs never mention it.  I  guess no one implement the interface in Symfony framework (we do it in BackBee CMS)

File: data/command-line.yml (hunk 2 above)
Comment (mickaelandrieu) [inline data/command-line.yml]: This assertion should be false :)  thanks!

File: data/command-line.yml (hunk 3 above)
Comment (mickaelandrieu) [inline data/command-line.yml]: Yes!  Thank you @xabbuh

**Comment by mickaelandrieu** [comment]
Thanks for review @xabbuh,  I will update the Set.  Also,  good luck if you are training for the certification: to me you are already a Symfony expert

**Comment by xabbuh** [comment]
Thank you @mickaelandrieu! :)

**Pull request merged**